source/Ship.h

```diff
@@ -455,9 +459,10 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Characteristics of the chassis:
 	bool isDefined = false;
 	const Ship *base = nullptr;
+	std::string trueName;
+	std::string variantTrueName;
```

Comment:
I don't think you need this variable. Ships should have one true name, which is what you refer to in the data files. If the ship is a variant, then `trueName` is the variant name. If not it's the same as the model name (but that's fine).

Reply:
That was just renamed `variantName`. I named it back but if I removed it altogether it wouldn't be available for logging or fleet creation with variant ships in them wouldn't be possible either. I thought that `variantTrueName` would be more descriptive since it's never shown for players but is just used as data.

Reply:
> I removed it altogether it wouldn't be available for logging and fleet creation with variant ships in them wouldn't be possible either

For logging we should always use `TrueName()`, which for variants would be their variant name. Fleet creation with variants ships should still be possible, because ships are referred to using their true name. Unless I'm missing something.

Reply:
I'd be more comfortable with leaving refactoring `VariantName` away to another PR and still keep its use as is. This one is about splitting the display model name as separate entity.